torch/csrc/autograd/init.cpp

```diff
@@ -12,23 +12,28 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
   using namespace torch::autograd::profiler;
   auto tensor_module = THPObjectPtr(PyImport_ImportModule("torch.tensor"));
   if (!tensor_module)
-    throw python_error();
+    return nullptr;
 
   // NOTE: "leaks" THPVariableClass
   THPVariableClass = PyObject_GetAttrString(tensor_module, "Tensor");
   if (!THPVariableClass)
-    throw python_error();
+    return nullptr;
```

Comment:
Are you sure `PyObject_GetAttrString` sets the error on failure? It is not mentioned in the doc like it is for ImportModule`.